daphne/src/roles_test.rs

```diff
@@ -299,3 +314,69 @@ async fn http_post_aggregate_abort_helper_state_overwritten() {
         assert_eq!(e, "unexpected message for aggregation job (already exists)")
     );
 }
+
+#[tokio::test]
+async fn http_post_upload_fail_send_invalid_report() {
+    let leader = MockAggregator::new();
+    let task_id = leader.nominal_task_id();
+
+    // Construct a report payload with a 0'd out task ID.
```

Comment:
It doesn't matter that it's 0'd out; what is relevant is that it's not the right task ID.

Reply:
Right. Updated comment.